AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -836,6 +836,9 @@ protected void onResume() {
             updateDeckList();
             setTitle(getResources().getString(R.string.app_name));
         }
+        /** Complete task and enqueue fetching nonessential data for
+         * startup. */
+        CollectionTask.launchCollectionTask(LOAD_LAZY_COLLECTION);
```

Comment:
Then this would become like 
```suggestion
        CollectionTask.launchCollectionTask(LOAD_COLLECTION_COMPLETE);
```

Reply:
Done